tools/schemacode/utils.py

```diff
@@ -132,3 +133,55 @@ def flatten_multiindexed_columns(df):
     df.index.name = "Entity"
     df = df.drop(columns=["DataType"])
     return df
+
+
+def _get_link(string):
```

Comment:
In `utils.py`, these make sense as public functions:

```suggestion
def get_link(string):
```